web/controllers/email_preview_controller.ex

```diff
@@ -9,11 +9,11 @@ defmodule Constable.EmailPreviewController do
       email_body(email_name) |> Repo.rollback
     end
 
-    html(conn, body)
+    text(conn, body)
   end
 
   defp email_body(email_name) do
-    email_for(String.to_atom(email_name)).html_body
+    email_for(String.to_atom(email_name)).text_body
```

Comment:
Was this intended to be a temporary change? Or do you prefer for the email preview to display the text version vs the html version?